src/models/People.js

```diff
@@ -78,7 +78,18 @@ static async displayPeopleProfessional() { // returns array of Timesheet Table
         {name: 'people_id', value:{longValue: userId}},
       ]);
 
-      const capabilities = rawCapabilities.records.map(([{ stringValue: capability }])=> capability);
+      const capabilities = rawCapabilities.records.reduce((acc, curr) => {
+        const [{ stringValue: capability },
+                { stringValue: params1 },
+                { stringValue: params2 },
+                { stringValue: params3 }] = curr;
+
+        return [...acc, 
+                ...(params1 ? [`${capability}(${params1})`] : []), 
+                ...(params2 ? [`${capability}(${params2})`] : []), 
+                ...(params3 ? [`${capability}(${params3})`] : [])]
```

Comment:
Must consider the case where if all the params are null, then include the single capability (backwards compatible).
```suggestion
        return [...acc, 
                ...(!params1 && !params2 && !params3 && [capability]),
                ...(params1 ? [`${capability}(${params1})`] : []), 
                ...(params2 ? [`${capability}(${params2})`] : []), 
                ...(params3 ? [`${capability}(${params3})`] : [])]
```